### PR TITLE
[WD-12238] new webpage at /data/opensearch/managed

### PR DESCRIPTION
**Pull request by akbarkz**
## Done

Added new webpage according to this [copy doc](https://docs.google.com/document/d/19gcIoKEibXTvZCsmtYCSQBMnBUmNUJI6RaLoMHL2_TA/edit) and this [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=166-4958&t=e5exMaJ2QIAaTFEt-0).

## QA

- Open webpage at https://canonical-com-1317.demos.haus/data/opensearch/managed and check if it complies with the copy doc and the design.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12238

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
all images are lacking [the vanilla highlighted background](https://vanillaframework.io/docs/examples/patterns/image/container/highlighted?theme=light)

**Comment by akbarkz** [comment]
@lyubomir-popov I addressed all your remarks, except one that is about having equal height for pricing cards. Could you please help me with that, cause using existing Vanilla utilities does not seem to be able to help.

**Comment by lyubomir-popov** [comment]
can you restart the demo pls - I get a 404

**Comment by lyubomir-popov** [comment]
- lines need to be as wide as the grey boxes:
image
- text unnecessarily compressed hjere, causing excessive wrapping. Can we give it one more column?
image

For the equal heights, since we don't have a component for this in vanilla (we only seem to use this on the u.c site - is that correct @Sophie-32 ?) maybe just hardcoded min-heights? This is what I get:

image

for the signpost logo, can we please use this file at 33px height, width set to auto:
https://assets.ubuntu.com/v1/ff455ff8-opensearch-logo--signpost.png

**Comment by akbarkz** [comment]
@lyubomir-popov thanks for the review!
I replaced the logo and fixed the layout for it, I noticed it was behaving weirdly for smaller screens. 
As for cards, I added min-height as you suggested. Overall it looks good now, but on some smaller screen sizes they still differ in height. So please have a look and see if we're fine with that.

**Comment by lyubomir-popov** [comment]
@akbarkz Just one question, this has a play icon, shouldn't it be clickable? 
image
The rest looks great!

File: templates/data/opensearch/managed.html [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+        <div class="p-section--shallow">
+          <h1>Managed OpenSearch by Canonical</h1>
+          <p>
+            Free up time to focus on building applications and business development rather than maintaining your database. Get automation, enterprise-grade security, scalability, availability, monitoring, and support from trusted experts with our Managed OpenSearch service.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="data-opensearch-modal">Schedule a call</a>
```
Comment (anthonydillon) [inline templates/data/opensearch/managed.html:39]: This button doesn't work

**Review by anthonydillon — APPROVED** [review]
Nice work @akbarkz, code looks good but one small comment about the CTA at the top of the page.

File: templates/data/opensearch/managed.html (hunk 1 above)
Comment (akbarkz) [reply templates/data/opensearch/managed.html:39]: you're right, sorry, totally missed that. Will push a fix. Thanks for the review!

**Comment by anthonydillon** [comment]
Should there be a link to this new page somewhere?

**Pull request merged**